feature/login/src/main/kotlin/register/NickNameViewModel.kt

```diff
@@ -0,0 +1,11 @@
+package register
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class NickNameViewModel: ViewModel() {
+    val name = MutableLiveData<String>()
```

Comment:
```suggestion
    val name = MutableLiveData<String>()
    val isClickable: LiveData<Boolean> = name.map { it. isNullOrBlank() }
```

LiveData 고유의 mapping 기능 활용하기(연산자)